### PR TITLE
Update Italian translation

### PR DESCRIPTION
**Pull request by Mek101**

### CODE REVIEW

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -272,35 +272,35 @@
...
+    <string name="pesdk_sticker_button_color">Colore</string>
+    <string name="pesdk_sticker_button_delete">Cancella</string>
+    <string name="pesdk_sticker_button_bringToFront">Sposta davanti</string>
+    <string name="pesdk_sticker_button_straighten">Raddrizza</string>
+    <string name="pesdk_sticker_button_replace">Sostituisci</string>
+    <string name="pesdk_sticker_button_opacity">Opacità</string>
+    <string name="pesdk_sticker_button_contrast">Contrasto</string>
+    <string name="pesdk_sticker_button_saturation">Saturazione</string>
+    <string name="pesdk_sticker_button_brightness">Luminosità</string>
     <string name="pesdk_sticker_category_name_custom">Uploads</string>
```
Comment (Mek101) [inline app/src/main/res/values-it/strings.xml:287]: @tibbi Would it be possible to have a bit of context for this line? I'm not sure what it means here

Comment (tibbi) [reply app/src/main/res/values-it/strings.xml:287]: sticker category which doesnt contain preset stickers, but custom ones added by the user

**Comment by Mek101** [comment]
Ready!

**Comment by tibbi** [comment]
nice, thanks

**Pull request merged**